### PR TITLE
[JAVA API][ARM CPU] Add COCO Android ARM demo (update old version)

### PR DESCRIPTION
**Pull request by allnes**
Update demo for Android ARM devices

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: modules/android_demos/coco_detection_android_demo/app/src/main/java/org/intel/openvino/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+        createRandomColor(COCO_CLASSES_91.length);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        try {
+            System.loadLibrary(OPENCV_LIBRARY_NAME);
+            System.loadLibrary(Core.NATIVE_LIBRARY_NAME);
```
Comment (likholat) [inline modules/android_demos/coco_detection_android_demo/app/src/main/java/org/intel/openvino/MainActivity.java:228]: Now we're loading native OV libs in OV Java bindings, so we don't need to do it here:
https://github.com/openvinotoolkit/openvino_contrib/blob/7239f8201bf18d953298966afd9161cff50b2d38/modules/java_api/src/main/java/org/intel/openvino/NativeLibrary.java#L16

File: modules/android_demos/coco_detection_android_demo/app/src/main/java/org/intel/openvino/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+        } catch (UnsatisfiedLinkError e) {
+            Log.e(
+                    "UnsatisfiedLinkError",
+                    "Failed to load native OpenVINO libraries\n" + e.toString());
```
Comment (likholat) [inline modules/android_demos/coco_detection_android_demo/app/src/main/java/org/intel/openvino/MainActivity.java:232]:
```suggestion
                    "Failed to load native OpenCV libraries\n" + e.toString());
```

**Review by likholat — CHANGES_REQUESTED** [review]

File: modules/android_demos/coco_detection_android_demo/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+- Choose and download [Android Studio](https://developer.android.com/studio) on your PC.
+
+- Select "File -> Open", and import demo project in `$OPV_HOME_DIR/openvino_contrib/modules/android_demos/coco_detection_android_demo`.
+
+- Build and run demo
+
+> The first time when you run the demo application on your device, your need to grant camera permission. Then run it again.
+
+> To build the project correctly, you should write in OpenCV `build.gradle` file `kotlinOptions` parameter same as current project's `build.gradle` file
```
Comment (likholat) [inline modules/android_demos/coco_detection_android_demo/README.md:75]: Please describe expected demo output. Maybe we can also add a screenshot/gif.

File: modules/android_demos/coco_detection_android_demo/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+# Coco Detection Android Demo
+
+![Running result](https://user-images.githubusercontent.com/47499836/189129594-2634e176-5a5b-4051-b713-ae9574a8c3da.png)
+
+This is a object detection demo for Android ARM devices.
```
Comment (likholat) [inline modules/android_demos/coco_detection_android_demo/README.md:5]:
```suggestion
This demo showcases inference of Object Detection network on Android ARM devices using [OpenVINO Java API](https://github.com/openvinotoolkit/openvino_contrib/tree/7239f8201bf18d953298966afd9161cff50b2d38/modules/java_api).
```

**Pull request merged**